### PR TITLE
GSoC 2022 student blog for ESCAPE - OSSR

### PR DESCRIPTION
**Pull request by sudersan01**

### CODE REVIEW

**Comment by sudersan01** [comment]
This pull request contains the blog post for ESCAPE - OSSR for GSoC 2022 mid-term evaluation. 
mentored by @garciagenrique

**Comment by garciagenrique** [comment]
@sudersan01 could not add me as a reviewer. Nevertheless, everything is OK from my side.

File: _gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+project: ESCAPE-OSSR
+title: Repository as a Service
+author: Sudersan S
+date: 04.08.2022
+year: 2022
+layout: blog_post
+logo: OSSR-logo.png
+intro: |
+This blog entails my progress, learning and experience acquired during the GSoC period.
```
Comment (agheata) [inline _gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md:10]:
```suggestion
 This blog entails my progress, learning and experience acquired during the GSoC period.
```

Comment (agheata) [reply _gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md:10]: You need to add a space at least at the beginning of the intro to be interpreted correctly

File: _gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### Personal Experience
+
+It has been a great learning experience, and I look forward to making more significant contributions. I am grateful to my mentors for their support and guidance. <br />
+Project Repository - https://gitlab.in2p3.fr/escape2020/wp3/ossr-jupyterlab-extension <br />
```
Comment (agheata) [inline _gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md:33]:
```suggestion
Project [Repository](https://gitlab.in2p3.fr/escape2020/wp3/ossr-jupyterlab-extension) <br />
```

File: _gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Project Proposal - https://hepsoftwarefoundation.org/gsoc/2022/proposal_ESCAPE-repository-as-a-service.html
```
Comment (agheata) [inline _gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md:34]:
```suggestion
Project [Proposal](https://hepsoftwarefoundation.org/gsoc/2022/proposal_ESCAPE-repository-as-a-service.html)
```

Comment (agheata) [reply _gsocblogs/2022/blog_ESCAPE-OSSR_Sudersan.md:34]: Please add also links to your commits or pull requests in the project repo, as well as any other relevant material (like  links to meetings/presentations)

**Review by agheata — CHANGES_REQUESTED** [review]

**Comment by sudersan01** [comment]
@agheata I have made the changes and updated the blog as specified. pls verify

**Review by agheata — APPROVED** [review]

**Pull request merged**